docs/src/content/docs/it/guides/customization.mdx

```diff
@@ -0,0 +1,415 @@
+import FileTree from '../../../../components/file-tree.astro';
+
+Starlight fornisce stili e funzionalità predefiniti sensati, così puoi iniziare rapidamente senza bisogno di configurazione.
+Se vuoi iniziare a personalizzare l'aspetto del tuo sito Starlight, questa guida fa al caso tuo.
+
+## Aggiungi il tuo logo
+
+L'aggiunta di un logo personalizzato all'intestazione del sito è un modo rapido per aggiungere il tuo marchio individuale a un sito Starlight.
+
+1. Aggiungi il file immagine del tuo logo alla directory `src/assets/`:
+
+    <FileTree>
+
+   - src/
+     - assets/
+       - **my-logo.svg**
+     - content/
+   - astro.config.mjs
+
+   </FileTree>
+
+2. Aggiungi il percorso del tuo logo come opzione [`logo.src`](/it/reference/configuration/#logo) di Starlight in `astro.config.mjs`:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Documentazione con il mio logo',
+   			logo: {
+   				src: './src/assets/my-logo.svg',
+   			},
+   		}),
+   	],
+   });
+   ```
+
+Per impostazione predefinita, il logo verrà visualizzato accanto al `title` del tuo sito.
+Se l'immagine del tuo logo include già il titolo del sito, puoi nascondere visivamente il testo del titolo impostando l'opzione `replacesTitle`.
+Il testo del `title` verrà comunque incluso per gli screen reader in modo che l'intestazione rimanga accessibile.
+
+```js
+starlight({
+  title: 'Documentazione con il mio logo',
+  logo: {
+    src: './src/assets/my-logo.svg',
+    replacesTitle: true,
+  },
+}),
+```
+
+### Varianti del logo chiaro e scuro
+
+Puoi visualizzare diverse versioni del tuo logo in modalità chiara e scura.
+
+1. Aggiungi un file immagine per ciascuna variante a `src/assets/`:
+
+   <FileTree>
+
+   - src/
+     - assets/
+       - **light-logo.svg**
+       - **dark-logo.svg**
+     - content/
+   - astro.config.mjs
+
+   </FileTree>
+
+2. Aggiungi il percorso alle varianti del tuo logo come opzioni "light" e "dark" invece di "src" in "astro.config.mjs":
+
+   ```js
+   starlight({
+     title: 'Documentazione con il mio logo',
+     logo: {
+       light: './src/assets/light-logo.svg',
+       dark: './src/assets/dark-logo.svg',
+     },
+   }),
+   ```
+
+## Layout della pagina
+
+Per impostazione predefinita, le pagine Starlight utilizzano un layout con una barra laterale di navigazione globale e un sommario che mostra le intestazioni della pagina corrente.
+
+Puoi applicare un layout di pagina più ampio senza barre laterali impostando [`template: splash`](/it/reference/frontmatter/#template) nel frontmatter di una pagina.
+Funziona particolarmente bene per le pagine di destinazione e puoi vederlo in azione sulla [home page di questo sito](/it/).
+
+```md
+---
+# src/content/docs/index.md
+
+tititletolo: La mia pagina di destinazione
+template: spruzzo
+---
+```
+
+## Sommario
```

Comment:
Ma in italiano si dice “sommario” oppure “indice dei contenuti” (o anche solo “indice”)? Guardando [Wikipedia](https://it.wikipedia.org/wiki/Indice_(editoria)), dice “chiamato impropriamente anche sommario” ma forse dipende dal contesto.

Reply:
Viene usato liberamente, probabilmente Wikipedia ha ragione sul fatto che "indice dei contenuti" sia quello corretto formalmente. Diciamo che "sommario" è più un riassunto, ragionandoci ora. Correggo.

Reply:
Grazie!